src/Microsoft.AspNetCore.Diagnostics.HealthChecks/HealthCheckMiddlewareOptions.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNetCore.Diagnostics.HealthChecks
     /// <summary>
     /// Contains options for the <see cref="HealthCheckMiddleware"/>.
     /// </summary>
-    public class HealthCheckOptions
+    public class HealthCheckMiddlewareOptions
```

Comment:
This is different from the scheme we use in other middleware e.g. https://github.com/aspnet/StaticFiles/blob/master/src/Microsoft.AspNetCore.StaticFiles/StaticFileOptions.cs. Intentional?

Reply:
Nevermind, just saw your comment. What other type is this conflicting with?

Reply:
Yeah this was kinda speculative on my part... will undo.